neon_stt_plugin_coqui/__init__.py

```diff
@@ -100,7 +146,17 @@ def convert_samplerate(self, audio, desired_sample_rate):
         return desired_sample_rate, np.frombuffer(output, np.int16)
 
 
-    def execute(self, audio, language=None):
+    def execute(self, audio):
```

Comment:
`language` should still be in the signature here to match the signature from the base class (https://github.com/OpenVoiceOS/OVOS-plugin-manager/blob/6e96a4a0913c75dd40215fe73b78d62c00b3562b/ovos_plugin_manager/templates/stt.py#L82)